ios/Classes/BetterPlayer.m

```diff
@@ -26,12 +27,10 @@ - (instancetype)initWithFrame:(CGRect)frame {
     _isInitialized = false;
     _isPlaying = false;
     _disposed = false;
+    _seekPosition = -1;
     _player = [[AVPlayer alloc] init];
     _player.actionAtItemEnd = AVPlayerActionAtItemEndNone;
-    ///Fix for loading large videos
-    if (@available(iOS 10.0, *)) {
-        _player.automaticallyWaitsToMinimizeStalling = false;
-    }
```

Comment:
- This seems this bug handles only mp4 file 
   - https://github.com/jhomlala/betterplayer/issues/224
- After removing it, I tested it with large video (> 5 hours), app works fine.